app/routes/document.js

```diff
@@ -11,6 +11,9 @@ export default class DocumentRoute extends Route {
     isSigning: {
       refreshModel: true,
       as: 'aanbieden_voor_handtekenen',
+    },
+    activeTab: {
+      refreshModel: false,
     }
   }
```

Comment:
1. we show `id??activeTab=versions` (or signatures). We can alias a dutch name for the left hand part.
2. now that we have this, we could replace the `isSigning` usage with `activeTab` with some tweaks to were it was used. Could be considered "out of scope"